webanno-support/src/main/java/de/tudarmstadt/ukp/clarin/webanno/support/AJAXDownload.java

```diff
@@ -101,9 +101,9 @@ protected String getFileName()
      */
     protected IResourceStream getResourceStream()
     {
-        
-        IResourceStream resStream = new AbstractResourceStream() {
-            private static final long serialVersionUID = 1L;
```

Comment:
Streams are hardly inspectable in a debugger - I don't think the variable adds anything here. You can leave it.